dbtmetabase/__init__.py

```diff
@@ -520,6 +525,7 @@ def models(
     metabase_use_http: bool = False,
     metabase_verify: Optional[str] = None,
     metabase_sync: bool = True,
+    metabase_sync_sources: bool = False,
```

Comment:
"Sync" only relates to polling for changes, not for the action of _exporting_ the models. Your change also changes the default behaviour, which will break the build for anyone expecting the sources to be exported.

The flag should be something along the lines of `exclude_sources` default to false and only NOT export sources when it's set to true.